### PR TITLE
merged associate segs into cut_tiles

### PR DESCRIPTION
**Pull request by gknisely**
@heffergm please review when you have a minute

### CODE REVIEW

File: scripts/cut_tiles.sh [hunk 1]
```diff
@@ -147,13 +184,9 @@ if  [ -n "$S3_PATH" ]; then
     #clean up s3 old files
     clean_s3 ${S3_PATH} 30
     #push up s3 new files
-    for f in ${cur_extras_dir}/*; do
-      aws s3 mv ${f} ${S3_PATH} --acl public-read
-    done
-    #signal other stacks to get new data
-    aws s3 ls ${S3_PATH}/planet_${stamp}.tar
+    aws s3 cp --recursive ${CUR_PLANET_DIR} ${S3_PATH} --acl public-read
```
Comment (gknisely) [inline scripts/cut_tiles.sh]: @heffergm is this logic correct?  Meaning copying an entire dir

Comment (heffergm) [reply scripts/cut_tiles.sh]: There shouldn't be any need to set the acl in the cp command, I've already made the buckets public. Otherwise looks fine.

**Comment by heffergm** [comment]
I gather that there are three or so S3 variables that are required to be set in the env but aren't provided with defaults at the head of the file? If I've got that right, maybe we could just mention them in comments somewhere up top?

Otherwise, let's :ship: :it: and we can get it running next week.

File: scripts/cut_tiles.sh (hunk 1 above)
Comment (heffergm) [reply scripts/cut_tiles.sh]: Also, if you need the directory name copied to S3 and not just the contents, you'll need something like: `aws s3 cp --recursive ${CUR_PLANET_DIR} ${S3_PATH}/${CUR_PLANET_DIR}`

Comment (gknisely) [reply scripts/cut_tiles.sh]: thank you

File: scripts/associate_segments.sh [hunk 2]
```diff
@@ -1,114 +0,0 @@
-#!/bin/bash
-
-DATA_DIR="/data/valhalla"
-
-# some defaults, if needed.
-export REGION=${REGION:-"us-east-1"}
-export OSMLR_DIR=${OSMLR_DIR:-"${DATA_DIR}/osmlr"}
-export NUMBER_OF_THREADS=${NUMBER_OF_THREADS:-"4"}
-
```
Comment (heffergm) [inline scripts/associate_segments.sh:9]: in production, the nodes will use role based access control to access whatever resources. It’s better to assume that if there’s some need to run this locally for whatever reason that the person can just do `AWS_PROFILE=whatever some_command.sh`

**Comment by heffergm** [comment]
👍

**Pull request merged**